src/other/tables.jl

```diff
@@ -103,3 +103,12 @@ IteratorInterfaceExtensions.getiterator(df::AbstractDataFrame) =
     Tables.datavaluerows(Tables.columntable(df))
 IteratorInterfaceExtensions.isiterable(x::AbstractDataFrame) = true
 TableTraits.isiterabletable(x::AbstractDataFrame) = true
+
+function Tables.subset(df::AbstractDataFrame, inds; view::Union{Bool, Nothing}=nothing)
+    res = view === true ? DataFrames.view(df, inds, :) : df[inds, :]
```

Comment:
```suggestion
    res = view === true ? view(df, inds, :) : df[inds, :]
```
don't need the `DataFrames.` qualifier, right?

Reply:
you need it as `view` is a kwarg name in this function